MockKit/MockPumpManager.swift

```diff
@@ -122,7 +125,7 @@ public final class MockPumpManager: TestingPumpManager {
                                                          imageName: "exclamationmark.circle.fill",
                                                          state: .critical)
         }
-        else if state.reservoirUnitsRemaining == 0 {
+        else if state.reservoirUnitsRemaining <= 0 {
```

Comment:
[nit] Should we consider just making sure the `reservoirUnitsRemaining` is pinned at zero?

Reply:
yes, I was thinking this as well. I'll have a look at what happens.